etna/transforms/encoders/categorical.py

```diff
@@ -23,6 +25,13 @@ class ImputerMode(str, Enum):
     none = "none"
 
 
+class ImputerType(str, Enum):
```

Comment:
Why is it called `ImputerType`?

Reply:
`ImputerMode` for strategies modes and `ImputerType` for return types, why not?

Reply:
`ImputerMode` it is called because it is the logic for imputation strategy when we don't have a value for this category. I'm not sure that it is a good naming, but it can be explained.

An `ImputerType` name doesn't have a proper explanation, because it doesn't relate to imputation. For example, `ReturnType` could be better probably.